src/NexusMods.DataModel/Synchronizer/SynchronizerService.cs

```diff
@@ -203,4 +205,46 @@ private IObservable<LoadoutSynchronizerState> CreateStatusObservable(LoadoutId l
 
         return statusObservable;
     }
+    
+    private void ThrowIfMainBinaryInUse(Loadout.ReadOnly loadout)
+    {   
+        // Note(sewer):
+        // Problem: Game may already be running.
+        // Edge Cases: - User may have multiple copies of a given game running.
+        //             - Only on Windows.
+        // Solution: Check if EXE (primaryfile) is in use.
+        // Note: This doesn't account for CLI calls. I think that's fine; an external CLI user/caller
+        var game = loadout.InstallationInstance.GetGame() as AGame;
+        var primaryFile = game!.GetPrimaryFile(loadout.InstallationInstance.Store)
+            .Combine(loadout.InstallationInstance.LocationsRegister[LocationId.Game]);
+        if (IsFileInUse(primaryFile))
+            throw new ExecutableInUseException("Game's main executable file is in use.\n" +
+                                               "This is an indicator the game may have been started outside of the App; and therefore files may be in use.");
```

Comment:
Clarify what the consequence is for the game files being in use: 
Synchronize operation can't be executed safely while files are in use.